### ISSUE TITLE
Azure storage incompatibility

### ISSUE THREAD

**Issue by yzhong94** [Azure/azure-batch-cli-extensions#56]
I'm getting this error message from installation.
OS: MacOS 10.13.4
Shell type: bash

azure-batch-extensions 1.0.1 has requirement azure-storage<0.35,>=0.32, but you'll have azure-storage 0.36.0 which is incompatible.

**Comment by xingwu1** [comment]
@yzhong94 @JaieParker 
What version azure-cli and azure-batch-cli-extension do you use?
The latest version azure-cli should be 2.0.31, and azure-batch-cli-extension version should be 2.2.1. 
Please upgrade to latest cli and extension. 

However, there has a compatible issue between latest cli and batch-extensions, so you have to manually update dependency `azure-batch-extensions` to 1.1.1.  

I don't have MacOS on hand, but if you fresh install azure-cli in Ubuntu, you can run following command to update the component:
```
sudo /opt/az/bin/python3 -m pip install -U azure-batch-extensions==1.1.1 
```

If you run azure-cli on windows, the following command is used to update the component:
```
"C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe" -m pip install -U azure-batch-extensions==1.1.1 
```

The compatible issue should be fixed at next azure-cli release.

**Comment by xingwu1** [comment]
The compatible issue has been addressed by newer azure-cli (2.0.32).

I couldn't repro this issue with latest azure-cli and batch extension. Please reopen this issue.
